### PR TITLE
[ENH] Adds issue and PR templates free of any license

### PR DESCRIPTION
**Pull request by eurunuela**
Follows #93 as I started working on the PR on my own fork and not on the browser as it was the case before.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Expected Behavior
```
[the remark this answers is not in the document]
Comment (eurunuela) [reply .github/ISSUE_TEMPLATE.md]: Sure. I’ll add some comments to this one and we can add a general template too.

**Comment by eurunuela** [comment]
What do you think the general template should look like @smoia ?

File: .github/ISSUE_TEMPLATE_BUGS.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: '[BUG]'
```
Comment (smoia) [inline .github/ISSUE_TEMPLATE_BUGS.md]:
```suggestion

```

File: .github/ISSUE_TEMPLATE_FEATURE_REQUEST.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: '[ENH]'
```
Comment (smoia) [inline .github/ISSUE_TEMPLATE_FEATURE_REQUEST.md]:
```suggestion

```

File: .github/ISSUE_TEMPLATE_BUGS.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## Steps to Reproduce the Problem
+<!--- Briefly point out the steps we should take to reproduce the problem -->
+
+  1.
+  2.
+  3.
+
+## Specifications
+<!--- Point out the version of phys2bids you are running and your OS version -->
+
```
Comment (smoia) [inline .github/ISSUE_TEMPLATE_BUGS.md]:
```suggestion
    - Python version: 
```

File: .github/ISSUE_TEMPLATE_BUGS.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  - Version:
```
Comment (smoia) [inline .github/ISSUE_TEMPLATE_BUGS.md]:
```suggestion
  - phys2bids version:
```

**Pull request merged**